src/main/pages/workspace-admin/workspaces-troubleshooting.adoc

```diff
@@ -16,39 +16,42 @@ Failures to start a workspace may be caused by different factors:
 [id="bad-recipes"]
 == Bad Recipes
 
-When a workspace is being started, environment link:recipes.html[recipe] is sent to Docker or OpenShift API. Che server then listens to events provided by a particular infrastructure. In simple words, Che server expects a running container (Docker) or pod (OpenShift). If the infrastructure was unable to create and start a container/pod from the provided recipe, Che server fails to start an environment, and consequently workspace start fails.
+When a workspace is starting, the environment link:recipes.html[recipe] is sent to Docker or to the OpenShift API. The Che server then listens to events provided by a particular infrastructure. In simple words, the Che server expects a running container (Docker) or pod (OpenShift). If the infrastructure was unable to create and start a container or a pod from the provided recipe, the Che server fails to start an environment and consequently the starting of the workspace fails.
```

Comment:
remove the parenthesis for (Docker) -> say something like "container such as Docker " or "Docker container" . Also remove the parenthesis for (OpenShift) -> say something like "OpenShift pod"